vaccine-standby/assets/index.html

```diff
@@ -0,0 +1,132 @@
+       
+      <p>This data is meant to help ensure vaccinations are administered as quickly as possible as supply becomes available. With this template, your agency can create a standardized list of eligible
+      vaccine recipients. In exchange for sharing this information, residents are notified when it is their turn to receive the vaccine.</p>
+    </header>
+    <div role="main">
+      <section>
+        <div id="deploy-flow" style="display: none;" class="deploy-studio-flow">
+          <h3>Finish configuring your app</h3>
+          <p>Your app is deployed, but you need to finish configuring it to get it working. Click the button below to automatically run the following deployment steps:</p>
+          <ul>
+            <li>Deploy Vaccine Standby List Studio Flow to power SMS chatbot</li>
+            <li>Link your phone number (<strong class="phone-number"></strong>) with Studio Flow</li>
+          </ul>
+          <a class="button" onclick="setup(event)">Deploy Studio Flow</a>
+          <div class="loader" style="display: none;"></div>
+        </div>
+        <div id="flow-deployed" style="display: none;" class="deploy-studio-flow">
+          <h3><span class="checkmark">&#10003;</span> Your Studio Flow is deployed</h3>
+          <p>You can move on to test your vaccine standby list below or click the button to make changes to your Studio Flow.</p>
+          <a id="open-studio" class="button button-outline" href="" target="_blank">Edit Studio Flow</a>
+        </div>
+      </section>
+      <section>
+        <h3>Test your Standby List</h1>
+        <p>You can try out your Standby List by sending a text message to <strong class="phone-number"></strong>. If your account is in trial mode, you will need to send the SMS from <a href="https://www.twilio.com/docs/usage/tutorials/how-to-use-your-free-trial-account#verify-your-personal-phone-number" target="_blank">verified phone numbers</a>.</p>
+        <div id="password-form" style="display: none;">
+            <p>To protect sensitive data, a password is required to see responses. This was the admin password you set when launching your Quick Deploy app. Or it can be found in the <code>.env</code>file in the root of your application.</p>
+            <p id="login-error" style="color:red;"></p> 
+            <form>
+            <div>
+              <label for="password">Password:
+                <input id="password-input" type="password" name="password">
+              </label>
+            </div>
+            <button class="button" onclick="login(event)">View Responses</button>
+          </form>
+        </div>
+      </section>
+      <section id="test-standby-list">
+        <p class="auth-successful"><span class="checkmark">&#10003;</span> Authenticated successfully</p>
+        <p>As the chatbot is completed, you should see records recorded below, pulled from <a class="execution-logs-link" href="" target="_blank">Studio Flow execution</a> logs from the last 30 days:</p>
```

Comment:
`logs from the last 30 days` aren't these actually only the latest 100 logs? Or are studio executions in general  only retrained for 30 days as well?

Reply:
Added info on max 100 records.